### DISCUSSION TITLE
Ublock Origin Lite (request): Hide number of blocked requests on icon

### DISCUSSION THREAD

**Discussion by Nojevah** [uBlockOrigin/uBOL-home#63] [Q&A]
Hello, sorry if you don't take request.

I don't know if it's planned, but I hope there will be a setting to disable the badge on extension icon. I use another extension to check mail and the notification badge from uBlock let me think I have new mails. I know I can hide the icon (that's what I do right now) but like I often need to use complete filtering, I need to click on ublock icon quite often.

![image](https://github.com/uBlockOrigin/uBlock-issues/assets/7541249/b2ee85c2-9f67-4827-9546-20986b035571)

![image](https://github.com/uBlockOrigin/uBlock-issues/assets/7541249/a62233c2-11ba-4705-98b3-c5662274c010)

**Comment by garry-ut99** [comment]
Untick "Show the number of blocked requests on toolbar icon"

![ss](https://github.com/user-attachments/assets/d977fef2-0c15-4328-9cba-f58c38f8d49a)
